test/integration/unregistered_user_can_view_categories_of_items_test.rb

```diff
@@ -10,10 +10,28 @@ class UnregisteredUserCanViewCategoriesOfItemsTest < ActionDispatch::Integration
     category_two.items.create(title: "Chai Tea",      description: 'forthy deliciousness',       price: 4.00, image: "google.com")
     category_two.items.create(title: "Cappuccino", description: 'super forthy deliciousness', price: 4.00, image: "google.com")
 
-      visit category_path(category_two.id)
-        # When I visit "/CATEGORY_NAME"
-        # I see all items assigned to that category
-
+    visit category_path(category_two.id)
+      # When I visit "/CATEGORY_NAME"
+      # I see all items assigned to that category
+    # within("#item#{item1.id}") do
       assert page.has_content?("Chai Tea")
+    # end
+  end
+
+  test 'user can view categories index page and click on category' do
+    category_one = Category.create(name:'coffee')
+    category_one.items.create(title: "pour over", description: 'it taste really good', price: 4.00, image: "google.com")
+    category_one.items.create(title: "drip",      description: 'it taste good',         price: 4.00, image: "google.com")
```

Comment:
Line is too long. [121/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.